src/test/java/org/minions/devfund/mauricioz/linkedList/ContainsTest.java

```diff
@@ -0,0 +1,366 @@
+package org.minions.devfund.mauricioz.linkedList;
+
+import java.util.LinkedList;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ Source code recreated from a .class file by IntelliJ IDEA.
+ (powered by Fernflower decompiler)
+ */
+public class ContainsTest {
+    private static final int LIST1FIRSTVALUE = 4;
+    private static final int LIST1SECONDVALUE = 8;
+    private static final int LIST1THIRDVALUE = 3;
+    private static final int LIST1FOURTHVALUE = 12;
+    private static final int LIST1FIFTHVALUE = 6;
+    private static final int LIST2EXTRAVALUE1 = 5;
+    private static final int LIST2EXTRAVALUE2 = 7;
+    private static final int LIST2EXTRAVALUE3 = 9;
+    private static final int LIST2EXTRAVALUE4 = 2;
+
+    /**
+     * constructor.
+     */
+    public ContainsTest() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)